### PR TITLE
feature/code-analysis-gems

### PR DESCRIPTION
**Pull request by Rodrigo-Miranda-0**
#### Board:
* [Ticket #brN88o3r](https://trello.com/c/brN88o3r/24-code-analysis-tools)
---
#### Description:
* Added code analysis gems Rubocop, Reek, Rails Best Practices, Brakeman
---
#### Notes:
*
---
#### Tasks:
- [x] Rubocop
- [x] Reek
- [x] Rails Best Practices
- [x] Brakeman
---
#### Risk:
*
---
#### Preview:

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -36,9 +36,15 @@ gem "bootsnap", require: false
...
 
+
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'brakeman', require: false
+  gem 'rubocop', '~> 1.36', require: false
+  gem 'rubocop-rails', require: false
+  gem 'reek', require: false
+  gem 'rails_best_practices', '~> 1.23', '>= 1.23.1', require: false
```
Comment (juanmferreira93) [inline Gemfile]: do not forget to add specific versions for each gem here

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+inherit_from: .rubocop_todo.yml
+require: rubocop-rails
+AllCops:
+  NewCops: enable
```
Comment (enzofab91) [inline .rubocop.yml:4]: Add new line here.

File: Gemfile [hunk 3]
```diff
@@ -36,9 +36,15 @@ gem "bootsnap", require: false
...
 
+
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'brakeman', '~> 5.3.1', require: false
+  gem 'rubocop', '~> 1.36', require: false
+  gem 'rubocop-rails', '~> 2.16.1', require: false
+  gem 'reek', '~> 6.1.1', require: false
+  gem 'rails_best_practices', '~> 1.23', '>= 1.23.1', require: false
```
Comment (LeandroSoria2) [inline Gemfile]: I think all of this should be only in the development environment.

**Review by enzofab91 — APPROVED** [review]
LGTM 💯

**Review by LeandroSoria2 — APPROVED** [review]
LGTM

**Pull request merged**